site/cds_rdm/inspire_harvester/transform_entry.py

```diff
@@ -31,29 +32,44 @@ def _record(self):
 
     def _files(self):
         """Transformation of files."""
+        current_app.logger.info("Start transforming files.")
         files, errors = self.transformer.transform_files()
+
+        current_app.logger.info(
+            f"Files transformation finished. Resulting object: {files}."
+        )
+        current_app.logger.info(f"Files transformation errors: {errors}.")
```

Comment:
```suggestion
        current_app.logger.error(f"Files transformation errors: {errors}.")
```
here we could also trigger the partial success